src/util/color/colorpalette.h

```diff
@@ -0,0 +1,21 @@
+//
+// Created by Ferran Pujol Camins on 27/10/2019.
+//
+
+#pragma once
+
+#include <QColor>
+#include "QList"
```

Comment:
Wrong #include directive. The whole class seems to be unfinished by violating many best practice rules.